### PR TITLE
Updated libs

### PR DESCRIPTION
**Pull request by yuriy-yarosh**
Updated all the libs for #240  to the most recent versions, changed outputs a bit.
See PR comments.

### CODE REVIEW

File: libs/aws-load-balancer-controller/config.jsonnet [hunk 1]
```diff
@@ -1,31 +1,22 @@
 local config = import 'jsonnet/config.jsonnet';
```
Comment (yuriy-yarosh) [inline libs/aws-load-balancer-controller/config.jsonnet:2]: extracted the versions variable over here

File: libs/banzaicloud-bank-vaults/config.jsonnet [hunk 2]
```diff
@@ -9,7 +11,7 @@ config.new(
     {
       output: v.version,
       prefix: '^com\\.banzaicloud\\.vault\\..*',
-      crds: ['https://github.com/banzaicloud/bank-vaults/raw/'+v.tag+'/charts/vault-operator/crds/crd.yaml'],
+      crds: ['https://raw.githubusercontent.com/'+v.tag+'/charts/vault-operator/crds/crd.yaml'],
```
Comment (yuriy-yarosh) [inline libs/banzaicloud-bank-vaults/config.jsonnet:14]: This link was no longer viable, had to change to `raw.githubusercontent.com` instead.

File: libs/cluster-api-provider-aws/config.jsonnet [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+  { version: '1.5', tag: 'v1.5.2'},
+  { version: '2.0', tag: 'v2.0.2'},
+];
+
 config.new(
   name='cluster-api-provider-aws',
   specs=[
     {
-      output: 'v1.2.0',
+      output: v.version,
```
Comment (yuriy-yarosh) [inline libs/cluster-api-provider-aws/config.jsonnet:15]: Changed versioning in here from `v.1.2.0` output to `1.2`.

File: libs/cluster-api/config.jsonnet [hunk 4]
```diff
@@ -1,14 +1,22 @@
 local config = import 'jsonnet/config.jsonnet';
 
+local versions = [
```
Comment (yuriy-yarosh) [inline libs/cluster-api/config.jsonnet:3]: Extracted the versions variable.

File: libs/fluxcd/config.jsonnet [hunk 5]
```diff
@@ -87,5 +87,47 @@ config.new(
       ],
       localName: 'fluxcd',
     },
+    {
```
Comment (yuriy-yarosh) [inline libs/fluxcd/config.jsonnet:90]: It's a bit hard to extract a version table in here, probably not worth the effort without prior automation and update management.

**Comment by Duologic** [comment]
May I suggest creating separate PRs per lib?

**Comment by yuriy-yarosh** [comment]
@Duologic Sure, will split this up, if it'll help with the review.

It would be best to consolidate all the `config.jsonnet` file structure and the version matrices - automate the updates with a dedicated build task. Updating current versions just helped me with getting a grasp over general structure and styling inconsistencies. 

So, I'll be making some ammends to separate versions into a separate `version.json` files, if you don't mind.

**Pull request closed without merging**